benchmarks/user/advisor/roofline.py

```diff
@@ -65,21 +64,25 @@
 def roofline(name, project, scale, precision, mode, th):
     pd.options.display.max_rows = 20
 
-    log('Opening project...')
+    log(f'Opening project {project}...')
     project = advisor.open_project(str(project))
 
     if not project:
-        err('Could not open project %s.' % project)
+        err(f'Could not open project {project}.')
```

Comment:
What does `open_project` return if it fails to open the project? This error message might return something cryptic since `project` gets overwritten

Reply:
I was sure I had replied in this, not sure I cannot see my message. This is part of the advisor API, it should already be there 

https://www.intel.com/content/www/us/en/developer/articles/training/how-to-use-the-intel-advisor-python-api.html